app/controllers/points_controller.rb

```diff
@@ -11,14 +11,20 @@ def create
   end
 
   def index
-    @points = Tour.find(params[:tour_id]).points
+    tour = Tour.find(params[:tour_id])
+    @points = tour.points.includes(:image_blob).includes(:image_attachment)
```

Comment:
Unnecessary chained `.includes` it takes `*args` or "splat args" so `.includes(:image_blob, :image_attachment)` should be fine